public/wp-content/themes/glific/header.php

```diff
@@ -31,6 +31,8 @@
 		</a>
 		<div class="ml-auto d-xl-none">
 			<a class="bg-theme-primary font-heebo-bold px-10 py-4 fz-18 leading-27 text-white text-decoration-none ml-11 glific-button-border d-xl-none mr-5"
+				data-toggle="modal" \
```

Comment:
Typo, please self review once.
```suggestion
				data-toggle="modal"
```